slide update

M61ss opened the pull request:
Updated code blocks adding languages where it doesn't cause errors.

nbicocchi commented:
mmmhhh pretty massive update. Are you sure it does not cause errors in IntelliJ??

M61ss commented:
It doesn't cause errors (now I do another check for sure). There are only a few warnings... if you want, I can remove them.

nbicocchi commented:
I've seen some changes that usually cause warnings in Intellij. For example a single line of code annotated with ```java.
Usually you have to annotate a whole class or a function.

M61ss commented:
Yes, I am fixing those. While fixing, I find out an IntelliJ's function called "Hide problems in code fences". If you didn't know that, you can view that in Settings -> Languages & Frameworks -> Markdown -> "Show problems in code fences".  At the moment I am removing warnings as you required me, but if you agree to enable that setting and so ignore errors, I can revert changes that I am doing at any time.

M61ss commented:
I removed all errors, warnings and spellcheck-errors caused by code fences. While I was fixing problems about code fences, I fixed some typing errors that IntelliJ found out in slides' text.

nbicocchi commented:
Amazing! Thank you. I'm a bit nervous about merging :)
